src/core/client/admin/routes/Moderate/Queue/ForReviewQueueRoute/ForReviewQueueRoute.tsx

```diff
@@ -0,0 +1,292 @@
+  relay,
+  query,
+}) => {
+  const [disableLoadMore, setDisableLoadMore] = useState(false);
+
+  const markFlagged = useMutation(MarkFlagReviewedMutation);
+
+  const [{ moderationQueueSort }] = useLocal<ForReviewQueueRouteLocal>(graphql`
+    fragment ForReviewQueueRouteLocal on Local {
+      moderationQueueSort
+    }
+  `);
+
+  const [, isRefetching] = useRefetch<
+    ForReviewQueueRoutePaginationQueryVariables
+  >(relay, 5, {
+    orderBy: moderationQueueSort,
+  });
+
+  const loadMore = useCallback(() => {
+    if (!relay.hasMore() || relay.isLoading()) {
+      return;
+    }
+
+    setDisableLoadMore(true);
+
+    relay.loadMore(
+      10, // Fetch the next 10 feed items
+      (error: any) => {
+        setDisableLoadMore(false);
+
+        if (error) {
+          // eslint-disable-next-line no-console
+          console.error(error);
+        }
+      }
+    );
+  }, [relay]);
+
+  const onReview = useCallback(
+    async (id: string, reviewed: boolean) => {
+      await markFlagged({ id, reviewed });
+    },
+    [markFlagged]
+  );
+
+  if (!query || !query.viewer) {
+    return null;
+  }
+
+  if (isRefetching) {
+    return <LoadingQueue />;
+  }
+
+  const flagActions = query.flags.edges.map((edge: { node: any }) => edge.node);
+
+  return (
+    <IntersectionProvider>
+      <HorizontalGutter size="double">
+        <Table fullWidth className={styles.table}>
+          <TableHead>
+            <TableRow>
+              <TableCell>
+                <Localized id="forReview-time">Time</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-comment">Comment</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reportedBy">Reported by</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reason">Reason</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-description">Description</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reviewed">Reviewed</Localized>
+              </TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {flagActions.map((a, i) => (
+              <ForReviewQueueRow
+                key={a.id}
+                id={a.id}
+                comment={{ ...a.comment, revision: a.revision }}
+                username={a.flagger ? a.flagger.username : ""}
+                reason={a.reason}
+                additionalDetails={a.additionalDetails}
+                createdAt={a.createdAt}
+                reviewed={a.reviewed}
+                onReview={onReview}
+              />
+            ))}
+          </TableBody>
+        </Table>
+        {relay.isLoading() && (
+          <Flex justifyContent="center">
+            <Spinner />
+          </Flex>
+        )}
```

Comment:
This is not required, `<AutoLoadMore>`  already shows a spinner and I see double spinners with this.